### PR TITLE
[US1] Initialize Spring Boot application.

### PR DESCRIPTION
**Pull request by oleksandr-skyrda**
Social-network-api folder is created & Spring application is initialised & readme file is updated & .gitignore is modified

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+        </dependency>
+
+<!--        <dependency>-->
+<!--            <groupId>org.flywaydb</groupId>-->
+<!--            <artifactId>flyway-core</artifactId>-->
+<!--        </dependency>-->
```
Comment (miegami) [inline pom.xml]: Please, remove commented dependencies.

File: README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# FP3SocialNetwork - Twitter
```
Comment (miegami) [inline README.md]: Please, add details on how to run this app from the terminal to the readme.

**Comment by miegami** [comment]
Add folder for be.

**Comment by miegami** [comment]
Please, rename the pull request using the story number and name.

File: pom.xml (hunk 1 above)
Comment (oleksandr-skyrda) [reply pom.xml]: Removed commented dependencies.

**Comment by oleksandr-skyrda** [comment]
> Add folder for be.

Folder created

File: README.md (hunk 2 above)
Comment (oleksandr-skyrda) [reply README.md]: Added description of launching the application from the command line

> Please, add details on how to run this app from the terminal to the readme.

**Comment by oleksandr-skyrda** [comment]
> Please, rename the pull request using the story number and name.

Pool request renamed

**Review by miegami — APPROVED** [review]

**Review by ol-skyrda — APPROVED** [review]

**Pull request merged**